ui/src/views/Clients/ViewClients/index.jsx

```diff
@@ -12,16 +13,20 @@ import Button from '../../../components/Button';
 import ClientsTable from '../../../components/ClientsTable';
 import { VIEW_CLIENTS_PAGE_SIZE } from '../../../utils/constants';
 import clientsQuery from './clients.graphql';
-import { withAuth } from '../../../utils/Auth';
 import ErrorPanel from '../../../components/ErrorPanel';
 
 @hot(module)
-@withAuth
 @graphql(clientsQuery, {
   options: props => ({
     variables: {
       clientOptions: {
-        ...(props.user ? { prefix: props.user.credentials.clientId } : null),
+        ...(props.history.location.search
+          ? {
+              prefix: decodeURIComponent(
+                parse(props.history.location.search)['?search']
```

Comment:
I think something like the following would also work:

https://github.com/taskcluster/taskcluster/blob/0d6377a32c874be65929b36149d13eaed8dfd5d1/ui/src/views/PulseMessages/index.jsx#L38

Reply:
> Interesting. It seems to happen whether we await or not. I can try to investigate. Any ideas?

I believe what's happening is loading is stuck on true. Here's a relevant issue: https://github.com/apollographql/apollo-client/issues/1186

A proposed solution is adding notifyOnNetworkStatusChange: true, which I've tested by adding it to the graphql object at the start. Still doesn't seem to fix it.

Await on the refetch, however, does seem to prevent this. Shouldn't we keep it in?

Just to confirm - how I'm reproducing the issue:

1. Visit http://localhost:5080/auth/clients
2. Search with improper string (e.g. 'sadas sads')
3. Search again with empty string '',

Reply:
If awaiting fixes it, then let's keep it :)